api-reference/user-app/notifications/notification.yml

```diff
@@ -10,63 +10,285 @@ servers:
+            token_uri:
+              description: The URI used to obtain the token.
+              type: string
+            auth_provider_x509_cert_url:
+              description: The URL for the auth provider's X.509 certificate.
+              type: string
+            client_x509_cert_url:
+              description: The URL for the client's X.509 certificate.
+              type: string
+            universal_domain:
+              description: The universal domain associated with the service account.
+              type: string
+      description: >-
+        The request body must contain the FCM payload information.
+        The format should be:
+
+        ```json
+        {
+          "sa_key": {
+            "type": "string",
+            "project_id": "string",
+            "private_key_id": "string",
+            "private_key": "string",
+            "client_email": "string",
+            "client_id": "string",
+            "auth_uri": "string",
+            "token_uri": "string",
+            "auth_provider_x509_cert_url": "string",
+            "client_x509_cert_url": "string",
+            "universal_domain": "string"
+          }
+        }
+        ```
+      example:
+        sa_key:
+          type: service_account
+          project_id: example-project
+          private_key_id: 1234567890abcdef
+          private_key: -----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASC...\n-----END PRIVATE KEY-----\n
+          client_email: example@example.iam.gserviceaccount.com
+          client_id: 12345678901234567890
+          auth_uri: https://accounts.google.com/o/oauth2/auth
+          token_uri: https://oauth2.googleapis.com/token
+          auth_provider_x509_cert_url: https://www.googleapis.com/oauth2/v1/certs
+          client_x509_cert_url: https://www.googleapis.com/robot/v1/metadata/x509/example@example.iam.gserviceaccount.com
+          universal_domain: example.com
 paths:
-  /profile/add-device:
+  /notification/add-device:
     post:
       tags:
         - Notifications
-      summary: add-device
+      summary: Add Device
+      description: Call to add or register a device for notifications.
       requestBody:
+        required: true
         content:
           application/json:
             schema:
-              type: object
-              example:
-                devicePayload:
-                  app-id: ''
-                  app-platform: android
-                  app-version: '1571'
-                  did: aa8639d3ff050d55
-                  fcm-id: fr3Ybq-2QHW_JptUCHwhkh:APA91bEHF015uSYgjNSnCyC-RGbGmmHfqARZpi6eAWwWCw3SWnLMEFu34w5pgd9uM-NFCkK3nuCivqQlqUE2jt7V6x_BNB4tCLucU7eNYHfX5PliniucZf0_4gZ4H2gJMZrb6JS6tAZZ
-                  fid: fr3Ybq-2QHW_JptUCHwhkh
-                  lang: en
-                  os-version: 30
-                  user-hash: ''
+              $ref: '#/components/schemas/AddDeviceRequest'
       parameters:
-        - name: Content-Type
+        - name: auth
```

Comment:
instead of auth header, let's use authorization like this -> https://github.com/eka-care/eka-docs/blob/62bf018980bd90dfac00934c459d3e1996737616/api-reference/user-app/records/records.yaml#L67

Reply:
done, add a security scheme